deploy/kessel-relations.yaml

```diff
@@ -159,3 +159,9 @@ parameters:
   - description: Image Tag
     name: RELATIONS_IMAGE_TAG
     value: latest
+  - description: Number of pods for spiceDB service
+    name: SPICEDB_REPLICAS
```

Comment:
The variables are defined, but I don't see them being used like with stage we have replicas set https://github.com/project-kessel/relations-api/blob/main/deploy/kessel-relations-deploy.yaml#L13
and https://github.com/project-kessel/relations-api/blob/main/deploy/kessel-relations-deploy.yaml#L42

Reply:
good catch! let me fix it

Reply:
fixed